services/airlock/docker-compose.yaml

```diff
@@ -27,11 +24,32 @@ services:
       AIRLOCK_WORKSPACE_DIR: /workspaces/
       AIRLOCK_REQUEST_DIR: /requests/
       DJANGO_DEBUG: "False"
-    command: gunicorn --config gunicorn.conf.py --certfile /certs/airlock.crt --keyfile /certs/airlock.key airlock.wsgi
     volumes:
       - /home/opensafely/airlock/workdir:/workdir
       - /srv/medium_privacy/workspaces:/workspaces:ro
       - /srv/medium_privacy/requests:/requests
       - ./certs:/certs
       # use our hardcoded DNS
       - /etc/hosts:/etc/hosts
+    
+  airlock:
+    # The version of docker-compose in Ubuntu 20.04 (1.25) does not support
+    # `extends`, so we use YAML anchor tricks to create a test service that
+    # extends the base service. This statement copies all the keys from the
+    # anchor &service above
+    <<: *service
+    container_name: airlock
+    ports:
+      - "443:8000"
+    command: gunicorn --config gunicorn.conf.py --certfile /certs/airlock.crt --keyfile /certs/airlock.key airlock.wsgi
+
+
+  airlock-file-uploader:
+    depends_on: [airlock]
+    # The version of docker-compose in Ubuntu 20.04 (1.25) does not support
```

Comment:
Ah, this comment, taken from the top of file is kind of outdated - my bad.

We're on 22.04 and we have docker compose v2, so we should be able to use extends.  We probably need to remove the version: "3.7"` line though, and also the comment? I think extends would be cleaner?

However, I just noticed that we are using `docker-compose` in the airlock justfile, which is incorrect, so maybe this is still the case :thinking:

Reply:
Sigh¸looks like I updated jobrunner to use docker compose v2, but not airlock in https://github.com/opensafely-core/backend-server/pull/253/files#diff-f0f1d6eea7a10be607fac2265f9905bee846de37baa1c62c06227219bd22abbe